mvrx-compose/gradle.properties

```
@@ -2,4 +2,4 @@ POM_NAME=Mavericks Compose
 POM_ARTIFACT_ID=mavericks-compose
 POM_PACKAGING=aar
 GROUP=com.airbnb.android
-VERSION_NAME=2.1.0-alpha02-SNAPSHOT
\ No newline at end of file
+VERSION_NAME=2.1.0-alpha03-SNAPSHOT
```

```suggestion
VERSION_NAME=2.4.0-alpha01-SNAPSHOT
```
The current mavericks-compose artifact should also be 2.3.0-alpha* not 2.1.0-alpha*